BangazonWorkforce/BangazonWorkforce.IntegrationTests/EmployeeTests.cs

```diff
@@ -51,7 +51,7 @@ public async Task Post_CreateAddsEmployee()
 
             string newFirstName = "FirstName-" + Guid.NewGuid().ToString();
             string newLastName = "LastName-" + Guid.NewGuid().ToString();
-            string isSupervisor = "true";
```

Comment:
Why did you change `isSupervisor` from `true` to `false`?

Reply:
I was testing to see if it worked with false. Just never changed it back